torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -305,72 +306,54 @@ def _wrap_fx_args_as_onnxscript_args(
     return onnxscript_args, onnxscript_kwargs
 
 
+@dataclass
+class FxOnnxInterpreterContext:
```

Comment:
No... Please don't do this. Let's make their signatures unchanged. As requested in other places, please just moving related functions into static methods of a class in this PR. If you make an explicit context instance, then it's similar to `self` and I am not ready for it.

Reply:
Using a `context` means we lose the hints of how individual components are passed and modified across different function boundaries. Therefore, readability decreases.

Reply:
removed context object